src/box/lua/config/source/file.lua

```diff
@@ -69,6 +69,18 @@ function methods.sync(self, config_module, _iconfig)
     assert(config_module._config_file ~= nil)
 
     local data = universal_read(config_module._config_file, 'config file')
+
+    do
```

Comment:
Minor: Could you clarify why `do ... end` is used here?

Reply:
I felt, that this code is alien to the place it was inserted, but now it seems like an overreaction, so removed.

Reply:
Well, honestly such splitting into blocks is widely used in Lua world, so I was just asking for your reasons... It was much better before, but if you feel it was not, I'll not insist.